### PR TITLE
Fix Checks results filters

### PR DESCRIPTION
**Pull request by jamie-suse**
# Description

Fixes changes to the new checks results filters from bd2f538.

Fixes [#1064](https://github.com/trento-project/web/pull/1064)

filters

## How was this tested?

Added new tests for filtering in `ExecutionResults.test.jsx`.

### CODE REVIEW

File: assets/js/components/ExecutionResults/ExecutionResults.jsx [hunk 1]
```diff
@@ -138,32 +136,57 @@ function ExecutionResults({
...
-                  }}
-                />
-              );
-            })}
+                return {
+                  checkID,
+                  error,
+                  expectations,
+                  failedExpectations,
+                  result: health,
```
Comment (arbulu89) [inline assets/js/components/ExecutionResults/ExecutionResults.jsx:149]: Why this change from `health` to `result`?
We use `health` all around the frontend code.
Is it because the `predicate` function expects that?

File: assets/js/components/ExecutionResults/ExecutionResults.test.jsx [hunk 2]
```diff
@@ -206,4 +206,88 @@ describe('ExecutionResults', () => {
...
+        catalog={catalog}
+        catalogError={error}
+        executionLoading={executionLoading}
+        executionData={executionData}
+        executionError={executionError}
+      />,
+      { route: `/clusters_new/${clusterID}/executions/last?health=passing` }
+    );
+
+    expect(screen.getByText('test-cluster')).toBeTruthy();
```
Comment (arbulu89) [inline assets/js/components/ExecutionResults/ExecutionResults.test.jsx:240]: I don't think we need all of these expects, they are already tested somewhere else.
I would only use the last 3 ones, which are the good ones for this scenario.
The same for the next test

**Review by arbulu89 — APPROVED** [review]
Hey @jamie-suse ,
The code looks great! Thank you for adding tests ;)
I would appreciate a gif to see it working

File: assets/js/components/ExecutionResults/ExecutionResults.jsx (hunk 1 above)
Comment (jamie-suse) [reply assets/js/components/ExecutionResults/ExecutionResults.jsx:149]: Yep that's correct, `predicate()` relies on `result` being present.

**Review by dottorblaster — APPROVED** [review]
LGTM
